tests/item_pickup_test.cpp

```diff
@@ -94,6 +91,38 @@ TEST_CASE( "putting items into inventory with put_in or i_add", "[pickup][invent
             }
         }
     }
+
+    WHEN( "avatar gets the backpack with pick_up" ) {
+        they.worn.clear();
+        // Construct locations including the backpack on the ground
+        item_location backpack_loc( map_cursor( ground ), &backpack_map );
+        const drop_locations &pack_droplocs = { std::make_pair( backpack_loc, 1 ) };
+        // With pick_up, the backpack gets a new item reference, but doesn't return it
+        they.pick_up( pack_droplocs );
```

Comment:
Since the avatar wears nothing, there is nothing to put the backpack in, so it doesn't get picked up. Pickup can't wield or wear items anymore, and even before it would have had to confirm a popup to do it.

Reply:
I suppose now would be a good time for me to add documentation to the `Character::pick_up` function explaining stuff like this :-)